taxi/forms.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, get_user_model
+from django.core.exceptions import ValidationError
+
+from .models import Car
+
+
+class DriverCreationForm(UserCreationForm):
+    first_name = forms.CharField()
+    last_name = forms.CharField()
+    email = forms.EmailField()
+
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "username",
+            "email", "password1", "password2",
+            "license_number",
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    LICENSE_NUMBER_LENGTH = 8
+
+    class Meta:
+        model = get_user_model()
```

Comment:
The `get_user_model()` function is used to access the `license_number` field. Ensure that the user model actually includes a `license_number` field, otherwise this will raise an error.